src/test/resources/features/addProductToCart.feature

```diff
@@ -0,0 +1,10 @@
+@mraicov
+Feature: Add product to Shopping cart
```

Comment:
please rename feature files to start with upper case, Feature files are an exception from the rule I've told you

Reply:
Done.
Thank You!